library/std/src/io/stdio.rs

```diff
@@ -712,26 +705,38 @@ impl Write for &Stdout {
     }
 }
 
+impl StdoutLock<'_> {
+    #[inline]
+    fn inner(&self) -> RefMut<'_, LineWriter<StdoutRaw>> {
+        #[cold]
+        fn init() -> LineWriter<StdoutRaw> {
+            LineWriter::new(stdout_raw())
+        }
+
+        RefMut::map(self.inner.borrow_mut(), |w| w.get_or_insert_with(init))
+    }
```

Comment:
I'm not convinced it's a good idea to move initialization of stdout into this method, which is part of the hot path for locked output. Why not keep it in `stdout()`?

Reply:
I've changed it back to using a `OnceLock`.